DateTimePicker/DateTimePicker.js

```diff
@@ -72,66 +49,40 @@ class DateTimePickerBase extends React.Component {
 		}
 	}
 
-	ranges = {
-		years: ['2018', '2019', '2020', '2021', '2022', '2023', '2024', '2025', '2026', '2027'],
-		months: ['1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12'],
-		days: ['1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12', '13', '14', '15', '16', '17', '18', '19', '20', '21', '22', '23', '24', '25', '26', '27', '28', '29', '30', '31'],
-		hours: ['12', '1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11'],
-		minutes: ['00', '01', '02', '03', '04', '05', '06', '07', '08', '09', '10', '11', '12', '13', '14', '15', '16', '17', '18', '19', '20', '21', '22', '23', '24', '25', '26', '27', '28', '29', '30', '31', '32', '33', '34', '35', '36', '37', '38', '39', '40', '41', '42', '43', '44', '45', '46', '47', '48', '49', '50', '51', '52', '53', '54', '55', '56', '57', '58', '59'],
-		meridiem: [$L('AM'), $L('PM')]
-	};
+	handleChange = (type) => ({value}) => {
```

Comment:
What for?

Reply:
I think I got confused with the onChange event. It would have been helpful if I had it on the DateTimePicker.

Reply:
This function was used in the previous implementation.